EIPS/eip-2537.md

```diff
@@ -343,42 +343,17 @@ Required properties for pairing operation:
-```
+## Reference Implementation
 
-## Implementation
-
-There is a various choice of existing implementations of the curve operations. It may require extra work to add an ABI:
-- BLS12-381 code bases for Eth 2.0 clients
-  - Chia's library in [C++](https://github.com/Chia-Network/bls-signatures)
-  - Milagro in [various languages](https://github.com/apache/incubator-milagro) 
-  - Noble in [TypeScript/Javascript](https://github.com/paulmillr/noble-bls12-381)
-- EIP1962 code bases with fixed parameters
-  - [Rust](https://github.com/matter-labs/eip1962)
-  - [Go](https://github.com/kilic/eip2537)
-  - [C++](https://github.com/matter-labs/eip1962_cpp)
-- Original implementation by zCash in [Rust](https://github.com/zcash/librustzcash/tree/master/pairing)
-- [MCL library](https://github.com/herumi/mcl) and it's bindings in other languages
+There are two fully spec compatible implementations:
+- One in Rust language that is based on the EIP1962 [code](https://github.com/matter-labs/eip1962). There exists an integration with OpenEthereum for this library
+- One implemented specifically for Geth as a part of the current [codebase](https://github.com/ethereum/go-ethereum/tree/master/crypto/bls12381)
 
```

Comment:
```suggestion
```
This should either contain an inline reference implementation, a link to a reference implementation in `../assets/eip-2537/*` or be removed.  External links and lists (which need to be maintained) should not be included in EIPs.